test/repositories/UniswapV3PoolRepository.test.ts

```diff
@@ -35,6 +40,10 @@ const pool2 = {
   token0: tokenB,
   token1: tokenA,
   chainId: ChainsIds.ETH,
+  liquidityActive: '2090386241959957974528',
+  liquidityLockedToken0: 27061.479326096796,
+  liquidityLockedToken1: 0.00006402189699766422,
```

Comment:
Javascript uses 64-bit floating point standard for the `number` type, giving at most 16 decimal digits of precision. I would decrease the decimal digits of this variable to at most 16 digits.

a = 0.123456789123456789123456789
a.toFixed(25)
'0.1234567891234567837965841'

Reply:
This is what we got from mongo, we don't need to use this value any place, it just query the biggest one. Maybe I need to change on `UniswapV3PoolRepository` because there I compare. But not sure if is necessary since it is just to know what is the biggest liquidityLockedToken0 or liquidityLockedToken1. 
Let me know

Reply:
I think it is not a big issue since it is just a test. We are not doing any calculation at all with that value. Just that it binary representation will not correspond to the string specified there.

Reply:
I decreased the decimal number. Thank you for flagging this